sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLog.scala

```diff
@@ -19,7 +19,9 @@ package org.apache.spark.sql.execution.streaming
 
 import java.io._
 import java.nio.charset.StandardCharsets
+import java.util.{Collections, LinkedHashMap}
```

Comment:
nit: can rename `LinkedHashMap` to `JLinkedHashMap` or others to make it clearer

Reply:
Yeah if there is a same class in Scala side, better to prefix it. Good point.

Reply:
sure